src/backend/TrafficCourts/Staff.Service/Controllers/DisputeController.cs

```diff
@@ -322,13 +323,13 @@ public async Task<IActionResult> GetDisputeAsync(long disputeId, CancellationTok
     [ProducesResponseType(StatusCodes.Status409Conflict)]
     [ProducesResponseType(StatusCodes.Status500InternalServerError)]
     [KeycloakAuthorize(Resources.Dispute, Scopes.Validate)]
-    public async Task<IActionResult> ValidateDisputeAsync(long disputeId, CancellationToken cancellationToken)
+    public async Task<IActionResult> ValidateDisputeAsync(long disputeId, Dispute? dispute, CancellationToken cancellationToken)
```

Comment:
Why are we passing in a dispute object? I thought the point spec-ing endpoints like this is to only pass in a disputeId and use that to retrieve the official copy from the database (and not use something someone may have modified and is different than the database version).

Reply:
This is a specific use case when the scanned ticket's fields parsed out correctly on staff portal but violation ticket object has not been saved into database properly because user did not click on the save button to update the violation ticket object with parsed form data. 

In this case, validate button is still active and if user directly clicks on the validate without saving it first, the data would never be saved properly in the database which creates other issues. So, this changes address that to get form data and save it on validate status change.

Reply:
Hmm. Okay.